packages/harpguru-core/src/utils/reduce-fullmatrix-to-viewablematrix/reduce-fullmatrix-to-viewablematrix.ts

```diff
@@ -0,0 +1,20 @@
+import type { HarpFaceMatrix } from 'harpparts'
+
+import { isPopulatedArray } from '../is-populated-array'
+import type { ColumnBounds } from '../../types'
+import { sliceMatrix } from '../../packages/slice-matrix'
+
+export const reduceFullMatrixToViewableMatrix = <T>(
+  activeMatrix: HarpFaceMatrix<T>,
```

Comment:
want to replace `active` wherever it's still relevant with `curr` to align with the now common use of `prev` and `next`

Reply:
Raised as techdebt at https://github.com/js-jslog/harpguru/issues/120#issuecomment-938443277